src/Microsoft.DotNet.Darc/Darc/Operations/UpdateSubscriptionOperation.cs

```diff
@@ -241,5 +330,40 @@ private bool UpdatingViaCommandLine()
            || _options.FailureNotificationTags != null
            || _options.SourceEnabled != null
            || _options.SourceDirectory != null
-           || _options.ExcludedAssets != null;
+           || _options.ExcludedAssets != null
+           || _options.StandardAutoMergePolicies != false
+           || _options.AllChecksSuccessfulMergePolicy != false
+           || _options.NoRequestedChangesMergePolicy != false
+           || _options.DontAutomergeDowngradesMergePolicy != false
+           || _options.ValidateCoherencyCheckMergePolicy != false
+           || _options.SourceFlowCheckMergePolicy != false;
+
+    private IEnumerable<string> GetExistingIgnoreChecks(MergePolicy mergePolicy) => mergePolicy
+                    .Properties
+                    .GetValueOrDefault(MergePolicyConstants.IgnoreChecksMergePolicyPropertyName)?
+                    .ToObject<IEnumerable<string>>()
+                    ?? [];
+
+    private void AddMergePolicyWithIgnoreChecksIfMissing(List<MergePolicy> mergePolicies, string policy)
```

Comment:
This method name feels a bit too descriptive. Maybe `AddMergePolicyWithDefaults`?

Also, what is trying to do exactly? It seems its trying to append ignore checks from a specific policy if it exists and if not then add a default policy which seems a bit too different of operations to combine into one function, no?